src/main/java/com/everis/utils/CarMessageListener.java

```diff
@@ -0,0 +1,44 @@
+package com.everis.utils;
+
+import javax.ejb.MessageDriven;
+import javax.inject.Inject;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+import javax.ejb.ActivationConfigProperty;
+
+
+@MessageDriven(activationConfig = {
+	    @ActivationConfigProperty(propertyName = "destinationLookup", propertyValue = "jms/destination"),
+	    @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Queue"),
+	    @ActivationConfigProperty(propertyName = "connectionFactoryLookup", propertyValue = "jms/CarsApp_API_ManagementQueue")
+	})
+public class CarMessageListener implements MessageListener {
+	
+	@Inject
+	CarService carService;
+	
+	
+	@Override
+	public void onMessage(Message message) {
+
+		try {
+		    System.out.println("Message received: " + message.getBody(String.class));
```

Comment:
It is not a good practice to use System.out to log messages. Switch to a logger implementation please.